features/cdmf-base-ui/org.wso2.carbon.devicemgt.ui.feature/src/main/resources/jaggeryapps/devicemgt/modules/user.js

```diff
@@ -180,6 +180,15 @@ var userModule = function () {
             } else {
                 var defaultUserClaims = privateMethods.buildDefaultUserClaims(firstname, lastname, emailAddress);
 
+                for (var role in userRoles){
```

Comment:
Acknowledge, Already fixed in moved code with commit https://github.com/wso2/carbon-device-mgt/commit/e537e2d841ca3c7946cd34e9da96f261e0ac94e1